src/extensions/registries/__tests__/page-registry.test.ts

```diff
@@ -72,15 +72,15 @@ describe("globalPageRegistry", () => {
 
   describe("getByPageMenuTarget", () => {
```

Comment:
The function name has changed
```suggestion
  describe("getByPageTarget", () => {
```